subtests/docker_cli/run_sigproxy/run_sigproxy.py

```diff
@@ -98,7 +112,7 @@ def initialize(self):
             self._init_container_attached(name)
         else:
             self._init_container_normal(name)
-        time.sleep(self.config['wait_start'])
+        self._wait_for_ready()
```

Comment:
Hmmm, actually in that case, what about adding it to the ``output`` module?  No unittest required - ``wait_for_output()`` is already tested, ``wait_for_ready()``  would just be wrapping it.  There are more than a handful of tests that could use this method.

Even if not, yep, your counter-proposal is perfectly fine as well.